portal-web/test/functional/com/liferay/portalweb/macros/Form.macro

```diff
@@ -4,6 +4,14 @@ definition {
 	var formName = "Created Form Name";
 	var rowNumber = "1";
 
+	macro addFieldToFieldsGroup {
+		DragAndDrop.javaScriptDragAndDropToBottom(
+			key_fieldLabel = "${targetFieldName}",
+			key_fieldName = "${sourceFieldLabel}",
```

Comment:
@wesleyroberts, we're assigning a field name to a field label and vice-versa, can you please make it consistent?

I mean:
```
key_fieldLabel = "${targetFieldLabel}",
key_fieldName = "${sourceFieldName}",
```

Reply:
changes made